airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -873,11 +875,11 @@ def process_executor_events(
                 ti.pid,
             )
 
-            if (active_ti_span := cls.active_spans.get(ti.key)) is not None:
+            if (active_ti_span := cls.active_spans.get("ti:" + ti.id)) is not None:
                 cls.set_ti_span_attrs(span=active_ti_span, state=state, ti=ti)
                 # End the span and remove it from the active_spans dict.
                 active_ti_span.end(end_time=datetime_to_nano(ti.end_date))
-                cls.active_spans.delete(ti.key)
+                cls.active_spans.delete("ti:" + ti.id)
```

Comment:
Elsewhere you have `"ti:" + str(ti.id)` -- we should be consistent. I don't mind which way we go, but lets be consistent please

Reply:
Fixed.